diff --git a/node_modules/paho-mqtt/paho-mqtt.js b/node_modules/paho-mqtt/paho-mqtt.js
index 9ca3d37..565940a 100644
--- a/node_modules/paho-mqtt/paho-mqtt.js
+++ b/node_modules/paho-mqtt/paho-mqtt.js
@@ -106,7 +106,7 @@ function onMessageArrived(message) {
 	/**
 	 * @private
 	 */
-	var localStorage = global.localStorage || (function () {
+	var localStorage = (function () {
 		var data = {};
 
 		return {
@@ -115,7 +115,13 @@ function onMessageArrived(message) {
 			removeItem: function (key) { delete data[key]; },
 		};
 	})();
-
+    try {
+        if ( global.localStorage ) {
+            localStorage = global.localStorage;
+        }
+    } catch ( e ) {
+        console.log( e );
+    }
 		/**
 	 * Unique message type identifiers, with associated
 	 * associated integer values.
